cmd/controller-manager/main.go

```diff
@@ -82,15 +84,16 @@ func main() {
 
 	itsClientLimits := clientopts.NewClientLimits[*pflag.FlagSet]("its", "accessing the ITS")
 	wdsClientLimits := clientopts.NewClientLimits[*pflag.FlagSet]("wds", "accessing the WDS")
+	processOpts.AddToFlags(pflag.CommandLine)
 	itsClientLimits.AddFlags(pflag.CommandLine)
 	wdsClientLimits.AddFlags(pflag.CommandLine)
 	klog.InitFlags(nil)
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
-	ctx := context.Background()
+	ctx, _ := ksctlr.InitialContext()
 	logger := klog.FromContext(ctx)
 	ctrl.SetLogger(logger)
```

Comment:
Are we going to get rid of this?

Reply:
Yes, when I am sure it is completely irrelevant.
This PR does not remove _all_ use of controller-runtime, that is blocked on https://github.com/kubestellar/kubeflex/issues/277